keyboards/k_type/config.h

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2015 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#define PREVENT_STUCK_MODIFIERS
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1c11
```

Comment:
It sounds like there may be an unresolved question around this vendor id, since I believe this one belongs to Input Club

Reply:
@haata would you mind us using this value as the default in QMK, or rather us choose our own? I realise there might not be much authority in an answer :)

Reply:
It would also be nice to do this for the whitefox... One thing worth noting, if we use a different value, the official I:C configurator doesn't recognize the keyboard until it's put into flash mode, but that might not be as important if they're using qmk.

Reply:
While running your own firmware, you can use which-ever you like.

We key off of a bunch of different things in the I:C configurator (descriptor information and such). I definitely fill out as much as possible in the USB descriptors in that regard, lol.
My ask is that you just indicate somewhere in the USB descriptor that the keyboard is running QMK (say in the iManufacturer or iProduct).

@jbondeson handles the I:C configurator so he knows exactly what we key off of. This kind of thing may change a bit over time as we figure out the best way to ID keyboards.

Reply:
This is how it shows up on OSX when running QMK.

```
            K-Type/QMK:

              Product ID: 0xb04d
              Vendor ID: 0x1c11
              Version: 2.a1
              Serial Number: 0xDEADBEEF
              Speed: Up to 12 Mb/sec
              Manufacturer: Input Club
              Location ID: 0x14420000 / 19
              Current Available (mA): 500
              Current Required (mA): 100
              Extra Operating Current (mA): 0
```

Reply:
Bah

```bash
lsusb -d 1c11: -vvv
```